src/main/java/com/redhat/quarkus/mandrel/collector/report/model/ImageStatsCollection.java

```diff
@@ -133,8 +154,30 @@ public ImageStats updateRunnerInfo(long id, RunnerInfo info) {
             return null;
         }
         if (info != null) {
+            em.persist(info);
             stat.setRunnerInfo(info);
         }
         return stat;
     }
+
+    // RunnerInfo
+
+    public RunnerInfo getSingleRunnerInfo(long id) {
+        return em.find(RunnerInfo.class, id);
+    }
+
+    @Transactional
+    public RunnerInfo add(RunnerInfo runnerInfo) {
+        em.persist(runnerInfo);
+        return runnerInfo;
+    }
+
+    // TODO: What to do with the stats? We leave them
+    // without runner info? Delete them too?
```

Comment:
Please remove that TODO. Or change to a proper coment. IMO Runner info is optional and code should account for that.

Reply:
Ack.